src/main/resources/templates/search-result.html

```diff
@@ -22,11 +22,11 @@ <h4>Search Result</h1>
 			<tr />
 		<thead />
 		<tbody>
-<!-- 		<tr th:each="theFaculty : ${faculties}"> -->
+		<tr>
```

Comment:
Here the html threw errors in eclipse on and my browser.  The table should have cells within rows for organization.